config_hub/finetune/README.md

```diff
@@ -3,66 +3,67 @@
 The table below lists the performances you can expect from the provided config files. Note that you can achieve lower memory consumption by lowering the micro batch size as needed. In addition, you can lower the rank (`lora_r`) in the LoRA configuration files and disable LoRA for certain layers (for example, setting `lora_projection` and other LoRA layer-specific parameters to `false`).
 For more information, see the [Dealing with out-of-memory (OOM) errors](../../tutorials/oom.md) on lowering the memory requirements.
 The "Cost" column refers to the on-demand compute cost on [Lightning AI](https://lightning.ai) where these benchmarks were executed.
+All experiments used the Alpaca2k dataset. The "Multitask score" refers to MMLU.
```

Comment:
> Also removes the "Dataset" and "Precision" columns as they are constant, to make space. 

> Also removes the "Dataset" and "Precision" columns as they are constant, to make space. 

It's obvious what these are if someone looks at the config files. But to make it easier on newcomers to this library, you could add it to the text. E.g., 

```suggestion
All experiments were conducted using bfloat-16 precision on the Alpaca2k dataset. The "Multitask score" refers to MMLU.
```

It's obvious what these are if someone looks at the config files. But to make it easier on newcomers to this library, you could add it to the text. E.g., "All experiments were conducted using bfloat-16 precision on the Alpaca2k dataset."